docs/sphinx/contributing/release_procedure.rst

```diff
@@ -51,6 +51,12 @@ are completed to avoid confusion.
      *   Update collaborators
      *   Update grant information
 
+#. Change the hpxMP release branch to be the most current release tag available
+   in the ``git_external`` hpxMP section of the main ``CMakeLists.txt``. Please
+   contact the maintainers at the `hpxMP repository
+   <https://github.com/STEllAR-GROUP/hpxMP>`_ to generate a new release to
+   synchronize with the HPX relelase.
```

Comment:
Small typo: relelease.

Reply:
Thanks, fixed.
You might want to fix this typo for Apex release under this file as well.